_includes/header.html

```diff
@@ -12,14 +12,10 @@ <h3 class="site-title">
 
     <nav class="nav" role="navigation" aria-label="Primary Menu">
       <ul>
-      {% assign current = page.url | split: '/' %}
-      {% for page in site.pages %}
-        {% unless page.url contains '/events/page' %}
-          {% if page.title and page.title != 'Page Not Found' %}
-            <li><a class="nav-item {% if page.url contains current[1] %}active{% endif %}" href="{{ page.url | prepend: site.baseurl }}">{{ page.title }}</a></li>
-          {% endif %}
-        {% endunless %}
-      {% endfor %}
+        <li><a class="nav-item" href="/about">about</a></li>
```

Comment:
@slowmanchan This change gets rid of the `active` class and requires us to update page items in multiple places. I think it'd be worth looking at a weighted ordering solution: http://stackoverflow.com/questions/13266369/how-to-change-the-default-order-pages-in-jekyll